shell/platform/darwin/ios/framework/Source/FlutterAppDelegate.mm

```diff
@@ -210,6 +210,32 @@ - (void)application:(UIApplication*)application
   }
 }
 
+- (void)application:(UIApplication*)application
```

Comment:
Nit: Has `clang-format` been applied to this patch?

Reply:
Whoops, nope. Will do that now.